manager/api/tests/test_jira.py

```diff
@@ -311,6 +316,27 @@ def test_needed_parameters(self):
 
         mock_jira_patcher.stop()
 
+    @patch("requests.get")
+    @patch("requests.put")
```

Comment:
Not an obligation, but for the sake of standardization could you use the format as on the `test_add_comment` function (without fixtures)? Again, not an obligation.

Reply:
Sorry, just saw @pothiers review... One sec, I'm looking at the comments.

Reply:
ok, so this one of those things that can be or cannot be jaja. I think is up to the developer, but the thing is (IIRC) this would be the first place on this repo that uses the fixtures format. Again, not an obligation, up to you ;). **But**, we should standardize this at some point and add some details to the `README.md` to point into a standard direction.